src/screens/SendSheet.tsx

```diff
@@ -170,7 +170,7 @@ export default function SendSheet() {
   const [debouncedRecipient] = useDebounce(recipient, 500);
 
   const [isValidAddress, setIsValidAddress] = useState(!!recipientOverride);
-  const [currentProvider, setCurrentProvider] = useState<StaticJsonRpcProvider | undefined>();
+  const [currentProvider, setCurrentProvider] = useState<StaticJsonRpcProvider | undefined>(getProvider({ chainId: ChainId.mainnet }));
```

Comment:
Provider now defaulting to mainnet. Before it was undefined (breaking ledger mainnet sends)